src/main/java/net/minecraftforge/fml/relauncher/IFMLLoadingPlugin.java

```diff
@@ -51,6 +51,8 @@ public interface IFMLLoadingPlugin
      * sorting behaviour.
      * All other mod behaviours are available however- this container can receive and handle
      * normal loading events
+     *
+     * The {@return} will be injected into {@link FMLInjectionData#containers}.
```

Comment:
Yeah.
If we want to just add a ModContainer, we will not be supported to add a empty IFMLLoadingPlugin.
https://github.com/CleanroomMC/Cleanroom/pull/25/files#diff-8d75c6a3f0546501feb61bf2dda9a467e43b851ee8f7e367cc8253f2ce2291e0